web/modules/custom/rsvplist/src/Form/RSVPForm.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace Drupal\rsvplist\Form;
+
+use Drupal;
```

Comment:
Non-namespaced classes/interfaces/traits should not be referenced with use statements

Reply:
then i will have to use the function of drupal core directly in my code with the namespace like this `\Drupal::some_function()`, but that is not acceptable as per vivek sir

Reply:
We need to use Dependency Injection to access the services